sql/core/src/main/scala/org/apache/spark/sql/sources/commands.scala

```diff
@@ -139,9 +140,8 @@ private[sql] case class InsertIntoHadoopFsRelation(
     }
 
     def writeRows(taskContext: TaskContext, iterator: Iterator[Row]): Unit = {
-      writerContainer.executorSideSetup(taskContext)
-
       try {
+        writerContainer.executorSideSetup(taskContext)
```

Comment:
oh, never mind. I did not get the change in `abortTask` when I made the comment.